Source/Foundation/bsfUtility/CMakeSources.cmake

```diff
@@ -180,18 +180,6 @@ set(BS_UTILITY_SRC_MATH
 	"bsfUtility/Math/BsLine2.cpp"
 )
 
-set(BS_UTILITY_INC_TESTING
-	"bsfUtility/Testing/BsTestSuite.h"
-	"bsfUtility/Testing/BsTestOutput.h"
-	"bsfUtility/Testing/BsConsoleTestOutput.h"
-)
-
-set(BS_UTILITY_SRC_TESTING
```

Comment:
Filters for old files were removed, but haven't been added for new ones. VS or XCode won't display those files in their workflow without them.